packages/@lwc/engine/src/framework/__tests__/html-element.spec.ts

```diff
@@ -8,6 +8,7 @@ import { compileTemplate } from 'test-utils';
 
 import { createElement, LightningElement } from '../main';
 import assertLogger from '../../shared/assert';
+import { registerDecorators } from '../decorators/register';
```

Comment:
We should be consistent and import `createElement`, `LightningElement`, and `registerDecorators` from the same path.